mongodb_communityserver/Executor.ps1

```diff
@@ -0,0 +1,15 @@
+﻿param (
+        [string]$extra,
+        [string]$localLogsDir
+    )
+
+$IncludePath = Join-Path -Path $PSScriptRoot -ChildPath "..\!include\Test.ps1"
+. $IncludePath
+
+$image = "mongodb/compass"
```

Comment:
This will fail the test. `-image` parameter is used in `StandardTest` to find the test folder to use and create the log with the name specified in `-image`.
See "Executor.ps1" in r-project_rtools for the correct example.

Reply:
updated the executor scripts to use a different method